mod-source-record-manager-server/src/main/java/org/folio/services/ChangeEngineServiceImpl.java

```diff
@@ -293,15 +293,15 @@ private boolean deleteMarcActionExists(JobExecution jobExecution) {
   }
 
   private boolean isCreateInstanceActionExists(JobExecution jobExecution) {
-    return containsCreateInstanceActionWithMatch(jobExecution.getJobProfileSnapshotWrapper());
+    return containsCreateInstanceActionWithNonMatch(jobExecution.getJobProfileSnapshotWrapper());
   }
 
-  private boolean containsCreateInstanceActionWithMatch(ProfileSnapshotWrapper profileSnapshot) {
+  private boolean containsCreateInstanceActionWithNonMatch(ProfileSnapshotWrapper profileSnapshot) {
```

Comment:
propose to leave the previous method name (containsCreateInstanceActionWithMatch) because "containsCreateInstanceActionWithNonMatch" says about action profil with NON-MAR "reactTo" value but actually the method will return `false` if childWrapper.getReactTo() = NON-MATCH, and will return `true` 
if childWrapper.getReactTo() = MATCH or childWrapper.getReactTo() = null

Reply:
right, also was considering about that

Reply:
thanks!